.github/workflows/docker-image.yml

```
@@ -0,0 +1,89 @@
+      - main
+
+jobs:
+  build:
+      runs-on: ubuntu-latest
+
+      steps:
+        - name: checkout code
+          uses: actions/checkout@v4
+
+        - name: Set up Golang 1.23
+          uses: actions/setup-go@v2
```

actionlint (1.7.7)

the runner of "actions/setup-go@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)